grpc/client/src/main/java/io/helidon/grpc/client/GrpcChannelDescriptor.java

```diff
@@ -70,6 +78,33 @@ public int port() {
         return port;
     }
 
+    /**
+     * Get the optional target string to use to resolve channel addresses.
+     *
+     * @return the optional target string to use to resolve channel addresses
+     */
+    public String target() {
```

Comment:
If this is optional, than you should return `Optional<String>` - Helidon APIs should not return (or accept) `null`.